packages/blade/package.json

```diff
@@ -289,7 +289,8 @@
     "plop": "3.1.1",
     "node-plop": "0.32.0",
     "svgson": "5.3.1",
-    "react-pdf": "9.2.1"
+    "react-pdf": "9.2.1",
+    "axios": "1.10.0"
```

Comment:
use native fetch

Reply:
global fetch has been there since node 18 🤔 this is the most basic usecase it should not fail like that. 

Let's see what's happening and try to see if possible to fix.